packages/core/src/components/_index.scss

```diff
@@ -94,11 +92,29 @@ ReactDOM.unmountComponentAtNode(myContainerElement);
 
 Check out the [React API docs](https://facebook.github.io/react/docs/react-api.html) for more details.
 
-Weight: 4
-
 Styleguide components.usage.vanilla
 */
 
+/*
```

Comment:
There's a section above in this file that says  "you need to include ambient types for ReactDOM..." which needs to be updated